include/can_frame.h

```diff
@@ -63,7 +63,7 @@
         (_frame).can_id += ((_value) & CAN_SFF_MASK);   \
     }
 
-#elif defined (__nuttx__) || defined (__FreeRTOS__) || defined (__MACH__) || defined (__WIN32__) || defined(__EMSCRIPTEN__) || defined(ESP_NONOS)
+#elif defined (__nuttx__) || defined (__FreeRTOS__) || defined (__MACH__) || defined (__WIN32__) || defined(__EMSCRIPTEN__) || defined(ESP_NONOS) || defined(ESP32)
```

Comment:
Will we not define __FreeRTOS__ as part of the Arduino build?  Would there not alternatively be a better Arduino specific define instead of an ESP32 specific define?

Reply:
I had started with FreeRTOS being defined but couldn't get it to compile cleanly. I'll take another pass at it.

Reply:
@bakerstu I've got it mostly compiling cleanly now with __FreeRTOS__ defined in platformio.ini but that doesn't solve it for Arduino IDE which I've not found a good solution for yet. I suspect we may need to add a few checks for ARDUINO_ESP32 at the top of certain files and define __FreeRTOS__ in those files.